src/pages/Doyun/Main/Contents.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import Feed from './components/Feed/Feed';
+import Rightmenu from './components/Rightmenu/Rightmenu';
+import './Contents.scss';
+
+export class Contents extends Component {
+  render() {
+    return (
+      <>
+        <main>
+          <Feed />
+          <Rightmenu />
+        </main>
+      </>
+    );
+  }
```

Comment:
```suggestion
    return (
      <main>
        <Feed />
        <AsideBar />
      </main>
    );
  }
```
- 불필요한 fragment가 있네요! 제거해주세요
- 방향이나 순서로 네이밍 하지 마시고 구체적인 역할이 드러나는 이름으로 컴포넌트나 변수명 지어주세요